### PR TITLE
Update gitignore

### PR DESCRIPTION
**Pull request by namangup**
Excluded all migration , pycache files.
Remove unnecessary __pycache__ and migration files.

### CODE REVIEW

**Comment by aashishpiitk** [comment]
> Excluded all migration , pycache files.
> Remove unnecessary **pycache** and migration files.

great idea

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 __pycache__/
-/campusdiscussbackend/__pycache__
+/**/__pycache__
 .vscode
+**/migrations/*
```
Comment (vikrantmalik051) [inline .gitignore]: Please add newline here.

**Comment by namangup** [comment]
@V1krant please have a look again.

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
 __pycache__/
-/campusdiscussbackend/__pycache__
+/**/__pycache__
```
Comment (V1krant) [inline .gitignore:2]: I think the first line would cover all files of the name `__pycache__`. So this might be unnecessary.

**Pull request merged**